### PR TITLE
Emit enabled inconsistency

### PR DESCRIPTION
**Pull request by prolic**
Resolves https://github.com/prooph/event-store/issues/366

### CODE REVIEW

File: src/Async/Projections/ProjectionsManager.php [hunk 1]
```diff
@@ -160,7 +160,7 @@ public function getQueryAsync(string $name, ?UserCredentials $userCredentials =
     public function updateQueryAsync(
         string $name,
         string $query,
-        bool $emitEnabled = false,
+        bool $emitEnabled = null,
```
Comment (enumag) [inline src/Async/Projections/ProjectionsManager.php]:
```suggestion
        ?bool $emitEnabled = null,
```

Null means no change I'm guessing?

File: src/Projections/ProjectionsManager.php [hunk 2]
```diff
@@ -144,7 +144,7 @@ public function getQuery(string $name, ?UserCredentials $userCredentials = null)
     public function updateQuery(
         string $name,
         string $query,
-        bool $emitEnabled = false,
+        bool $emitEnabled = null,
```
Comment (enumag) [inline src/Projections/ProjectionsManager.php]:
```suggestion
        ?bool $emitEnabled = null,
```

File: src/Async/Projections/ProjectionsManager.php (hunk 1 above)
Comment (prolic) [reply src/Async/Projections/ProjectionsManager.php]: yes

**Comment by prolic** [comment]
Thanks for your review @enumag

**Pull request merged**